view/adminhtml/layout/algolia_algoliasearch_analytics_index.xml

```diff
@@ -14,14 +14,14 @@
                     <argument name="default_selection_name" xsi:type="string" translate="true">Default Store View</argument>
                 </arguments>
             </block>
-            <block class="Magento\Backend\Block\Template" name="analytics.overview.form" template="Algolia_AlgoliaSearch::analytics/form.phtml">
+            <block name="analytics.overview.form" template="Algolia_AlgoliaSearch::analytics/form.phtml">
```

Comment:
@bsuravech ,  i suggest to restore class param and prevent issue with using block methods in template( escapeHtml, escapeJs, etc)

Reply:
Ah right on. Good catch @vmalyk!